src/sql/src/plan/statement/ddl.rs

```diff
@@ -4415,7 +4415,8 @@ generate_extracted_config!(
     (EnableEagerDeltaJoins, Option<bool>, Default(None)),
     (EnableNewOuterJoinLowering, Option<bool>, Default(None)),
     (EnableVariadicLeftJoinLowering, Option<bool>, Default(None)),
-    (EnableLetrecFixpointAnalysis, Option<bool>, Default(None))
+    (EnableLetrecFixpointAnalysis, Option<bool>, Default(None)),
+    (EnableJoinPrioritizeArranged, Option<bool>, Default(None))
```

Comment:
```suggestion
    (EnableJoinPrioritizeArranged, Option<bool>, Default(None)),
```
if the macro will allow it?

Reply:
It won't, unfortunately... (Btw., some of our similar macros also don't allow having a comma there, while some other of our macros don't allow _not_ having a comma there :sweat_smile: )